android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/EditTextFactory.java

```diff
@@ -412,7 +416,7 @@ private void attachRefreshLogic(Context context, JSONObject jsonObject, Material
         }
     }
 
-    private MaterialEditText getViewUsingAddress(String stepName, String fieldKey, JsonApi jsonApi) {
+    public MaterialEditText getViewUsingAddress(String stepName, String fieldKey, JsonApi jsonApi) {
```

Comment:
I see we've altered the access modifier here, do we intend to override it on the  implementations?

Reply:
Its used in mocking on the tests

Reply:
Ok, since the test is in a similar package you can give it a narrower scope and use **_protected_** instead. Also, annotate the method with **_@VisibleForTesting_**

Reply:
Yah this makes sense